Full featured explorer

dimakorzhovnik opened the pull request:
Transaction page #108

https://ipfs.io/ipfs/QmRvSA1arEL6Pg8YmEgUiqLVhnfZwBK1NMRRuaZTSXwZM2/#/txs/17D4BDCCCCC9ACE52CC2C47090517DBEDE7EA56A538411D59ED846C05B921F1B

dimakorzhovnik committed:
init txs

dimakorzhovnik committed:
fix

dimakorzhovnik committed:
txs link

dimakorzhovnik committed:
add error message

dimakorzhovnik committed:
fix error message

dimakorzhovnik committed:
init accont

dimakorzhovnik committed:
fix

dimakorzhovnik committed:
add pages AccountDetails

dimakorzhovnik committed:
fix account pages

dimakorzhovnik committed:
validator pages

dimakorzhovnik committed:
add Delegators, Delegated

dimakorzhovnik committed:
add actionBar to validatorsDetails

dimakorzhovnik committed:
send to AccountDetails

mastercyb reviewed (changes requested):
- [x] click on tx and account links open in the same window
- [x] add transactions to account page
- [x] move blocks into tabs on account page
- [x] add copypaste address on account page
- [x] account id center align
- [x] search by tx
- [x] search by account

dimakorzhovnik committed:
fix table Validator

dimakorzhovnik committed:
fix router

dimakorzhovnik committed:
[x] onClick tx, account links open in new window
[x] move blocks into tabs on account page

dimakorzhovnik committed:
[x] add copypaste address on account page

dimakorzhovnik committed:
move blocks into tabs on account page

dimakorzhovnik committed:
search by tx
search by account

dimakorzhovnik committed:
init

dimakorzhovnik committed:
fix conflicts

dimakorzhovnik committed:
fix

dimakorzhovnik committed:
transactions to account page

dimakorzhovnik committed:
add txs to validator pages

dimakorzhovnik committed:
fix link

dimakorzhovnik committed:
Remove filter for transactions

dimakorzhovnik committed:
fix  Same font on search result page

dimakorzhovnik committed:
fix brain

dimakorzhovnik committed:
Links for tabs

dimakorzhovnik committed:
Remove frame and header

dimakorzhovnik committed:
account id center align

dimakorzhovnik committed:
Align center tabs at heros and contracts page

dimakorzhovnik committed:
remove block height from cyberlinks and tx tabs

dimakorzhovnik committed:
TotalEUL - indicator in the top center

dimakorzhovnik committed:
Move unbonding to heros tab

dimakorzhovnik committed:
Add column to heros tab: Rewards

dimakorzhovnik committed:
fix

dimakorzhovnik committed:
add actionBar

dimakorzhovnik committed:
Claim rewards
action bar: cyberlink

dimakorzhovnik committed:
Infinite scrolling for cyberlinks and txs

dimakorzhovnik committed:
fix transaction

dimakorzhovnik committed:
fix validator pages

dimakorzhovnik committed:
fix URL paths

mastercyb reviewed (changes requested):
- [ ] Fix scroll (either full height or scroll with a content). [New issue](https://github.com/cybercongress/dot-cyber/issues/186)
- [ ] Mobile mockup: [Standalone epic](https://github.com/cybercongress/dot-cyber/issues/170)
- [x] Hide stages for unbonding under popup
- [x] Only moniker for validator
- [x] Font as in main
- [x] Several messages
- [x] Order of clolumns: status icon and tx id, timestamp and message (type), energy spent
- [ ] Handle several messages per tx. [New issue](https://github.com/cybercongress/dot-cyber/issues/185)
- [x] reward claim => Claim rewards
- [x] Verify for ownership for claim reward
- [ ] Balances by subscription. [New issue](https://github.com/cybercongress/dot-cyber/issues/187)
- [ ] Rewards like in oasis. [New issue](https://github.com/cybercongress/dot-cyber/issues/188)
- [x] Tab name Link => Cyberlinks
- [x] Add tab: Mentions. Table of cyberlinks to and from this contract

mastercyb reviewed (changes requested):
Hero page:
- [x] Above tabs: icon with moniker (link to site) in the middle. Percent of stake and uptime indicators
- [x] Tabs: Fans, Proofs, Main, Rumors, Leadership
- [x] Main: Table with parametrs (valoper address, contract address with link, staked EULs, commission rate, site, text, self delegation
- [x] Fans: Sorted table (contract, stake, share) of staker contracts by staked EULs
- [x] Burden: table of proposed and missed blocks, double sign evidence
- [x] Rumors: delegations and undelegations
- [x] Leadership - proposals and votes

dimakorzhovnik committed:
Only moniker for validator

dimakorzhovnik committed:
Font as in main

dimakorzhovnik committed:
Tab name Link => Cyberlinks

dimakorzhovnik committed:
reward claim => Claim rewards

dimakorzhovnik committed:
Order of clolumns

dimakorzhovnik committed:
Several messages

dimakorzhovnik committed:
Hide stages for unbonding under popup

dimakorzhovnik committed:
Mentions. Table of cyberlinks this contract

dimakorzhovnik committed:
merge

dimakorzhovnik committed:
fix  heroes pages

dimakorzhovnik committed:
Verify for ownsership for claim reward

dimakorzhovnik committed:
fix actionBar

dimakorzhovnik committed:
Rumors: delegations and undelegations
 Burden: table of proposed and missed blocks, double sign evidence
Tabs: Fans, Proofs, Main, Rumors, Leadership

mastercyb reviewed (changes requested):
- Add block page #179
- Fix hero page #178
- Add logic to gift page #124

dimakorzhovnik committed:
Leadership - proposals and votes

dimakorzhovnik committed:
Fans: Sorted table (contract, stake, share) of staker contracts by staked EULs

dimakorzhovnik committed:
Main: Table with parametrs

dimakorzhovnik committed:
Above tabs

mastercyb reviewed (changes requested):
Common:
- [x] pm UTC remove
- [x] fixed width for tabs

Hero:
- [x] main tab in the center
- [x] font size in main

dimakorzhovnik committed:
fix table tx UTC

dimakorzhovnik committed:
fix text table

dimakorzhovnik committed:
main tab in the center
 font size in main
 pm UTC remove
 fixed width for tabs

dimakorzhovnik committed:
fix actionBar

dimakorzhovnik committed:
Merge branch 'episode-1' into iss#108

dimakorzhovnik committed:
page block

dimakorzhovnik committed:
Merge branch 'iss#108' of https://github.com/cybercongress/dot-cyber into iss#108

dimakorzhovnik committed:
Move gift page back

dimakorzhovnik committed:
Cyberlinking on block and transaction page

dimakorzhovnik committed:
Link from transaction page to block page

dimakorzhovnik committed:
heroes.rumors: info about amount of delegate/undelegated

dimakorzhovnik committed:
show total eul in pocket

dimakorzhovnik committed:
add ATOMs and CYBs into pocket
 show total eul in pocket

dimakorzhovnik committed:
fix link

dimakorzhovnik committed:
fix table wallet
